deltacat/compute/compactor/steps/materialize.py

```diff
@@ -56,12 +67,25 @@ def materialize(
     read_kwargs_provider: Optional[ReadKwargsProvider] = None,
     s3_table_writer_kwargs: Optional[Dict[str, Any]] = None,
     deltacat_storage=unimplemented_deltacat_storage,
-) -> MaterializeResult:
+):
+    def _construct_delta_from_manifest_entry_reference_list(
+        manifest_entry_list_reference: List[ManifestEntry],
+    ) -> Delta:
+        manifest = Manifest.of(entries=manifest_entry_list_reference, uuid=str(uuid4()))
+        delta = Delta.of(
+            locator=DeltaLocator.of(partition.locator, None),
```

Comment:
Do we have to specify `None` explicitly when arg is optional? I think default is missing in the signature which may be forcing us to pass it.